addons/purchase_stock/models/product.py

```diff
@@ -130,8 +130,8 @@ def action_set_supplier(self):
             orderpoint.route_id = self.env['stock.rule'].search([('action', '=', 'buy')], limit=1).route_id.id
         orderpoint.supplier_id = self
         supplier_min_qty = self.product_uom._compute_quantity(self.min_qty, orderpoint.product_id.uom_id)
-        if orderpoint.qty_to_order < supplier_min_qty:
-            orderpoint.qty_to_order = supplier_min_qty
+        if orderpoint.product_min_qty < supplier_min_qty:
```

Comment:
```suggestion
        if orderpoint.qty_to_order < supplier_min_qty:
            orderpoint.qty_to_order = supplier_min_qty
        if orderpoint.product_min_qty < supplier_min_qty:
            orderpoint.product_min_qty = supplier_min_qty
```

Reply:
Hello @DjamelTouati 

thank you for your answer.
I have to disagree with keeping both.
Qty to order should always be computed by the orderpoints params (product_min_qty, max, multiple and its forecast)

For more details I updated my PR comment on top.